### ISSUE TITLE
Notebooks 2.0 // Frontend // Add test-and-build check to GitHub actions

### ISSUE THREAD

**Issue by jenny-s51** [kubeflow/notebooks#153]
Per our discussions regarding the a11y fixes in https://github.com/kubeflow/model-registry/pull/596 and the existing violations that were raised in https://github.com/kubeflow/dashboard/issues/64.

Following the existing infrastructure in Model Registry, we want to continue upholding and maintaining a11y standards and detect any violations in our CI checks throughout the Notebooks 2.0 effort. 

- [ ] Add a test-and-build check as done in https://github.com/ederign/model-registry/blob/main/.github/workflows/ui-frontend-build.yml
- [ ] Set up [Cypress UI testing](https://github.com/ederign/model-registry/commit/e9fbf2628291bd6a99d6f3f43465fe38751bb1d6) to detect any a11y violations in the frontend.

cc @ederign @paulovmr

**Comment by ederign** [comment]
Thank you so much @jenny-s51!

**Comment by paulovmr** [comment]
Commenting so I can be assigned :)

**Comment by ederign** [comment]
This issue is still planned for Notebooks 2.0.
